Dockerfile.rake

```diff
@@ -3,6 +3,9 @@ FROM public.ecr.aws/docker/library/ruby:2.7.7
 # Throw errors if Gemfile has been modified since Gemfile.lock
 RUN bundle config --global frozen 1
 
+# Updating packages
+RUN apt update && apt upgrade -yqq
```

Comment:
@jakekara do we want to say `apt-get` over `apt` across projects now? https://github.com/WPMedia/policeshootings/pull/592#pullrequestreview-1727678201

Reply:
Whichever works. Depends on the OS the base image is using which tools are installed. In this case, `apt` is fine.